### PR TITLE
hw-3

### PR DESCRIPTION
**Pull request by nixiieee**

### CODE REVIEW

File: notebooks/02-homework.ipynb
Comment (svkov) [inline notebooks/02-homework.ipynb:1]: зачем удаляешь ноутбук? если так смержить, то в итоговом репо не будет второго ноутбука.

сделай реверт и закоммить только 3 ноутбук

File: notebooks/03-homework.ipynb [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+    "from sklearn.metrics import accuracy_score\n",
+    "accuracy_score(y_test, y_pred_lr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79daf7c9-27d8-4434-bc3e-511308f2876a",
+   "metadata": {},
+   "source": [
+    "Обоснование выбора метрики: интуитивно понятная метрика, показывающая, какую долю объектов классификатор смог верно предсказать"
```
Comment (svkov) [inline notebooks/03-homework.ipynb]: Правда ли, что accuracy честная метрика на этих данных? Может ли быть ситуация, что какой-то класс мы вообще не учли, а accuracy будет высоким?

**Review by svkov — CHANGES_REQUESTED** [review]
С medium все ок, easy надо доделать

Ну и второй ноутбук надо спасти)

**Comment by nixiieee** [comment]
> С medium все ок, easy надо доделать
> 
> Ну и второй ноутбук надо спасти)

А что по поводу hard?

File: notebooks/02-homework.ipynb
Comment (nixiieee) [reply notebooks/02-homework.ipynb:1]: Его не получается удалить, потому что я случайно ответвилась от hw-2. Можно новую ветвь создать?

Comment (svkov) [reply notebooks/02-homework.ipynb:1]: Сделай так, чтобы в итоге второй ноутбук не потерялся)

**Comment by svkov** [comment]
> А что по поводу hard?

Не увидел)

Правда ли, что 5к картинок - достаточно?
Можно ли взять меньше? Если да, то на сколько?
5к картинок из каких классов надо взять? Есть ли классы, которым надо больше/меньше данных, чем остальным?

Ну и нужно написать код для ответа на эти вопросы

**Review by svkov — APPROVED** [review]
Супер! Первый зачтенный хард!
